tests/pipelines/regression/v1/test_batch_scoring_dag.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
```

Comment:
Mind adding these tests to `tests/pipelines/regression/v1/test_regression_pipeline.py` since they are from the same template?

Reply:
Given the special setup of this test suite (having a setup steps that creates the conditions that multiple steps check). I'm not sure that would work, unless we compile all the checks of this test file into one single test in that file. Would that be preferable?

Reply:
Can we at least change the file name to `test_regression_pipeline_batch_scoring.py` then?